conductor/lib/worker.py

```diff
@@ -496,3 +510,30 @@ def worker_queue_status_text(self):
             msg += '\t\t%s threads' % num_active_threads
             msg += '\n'
         return msg
+
+
+def empty_queue(queue):
+    '''
+    Remove and return all items from the given Queue object
+    '''
+    items = []
+
+    while True:
+        item = safe_get(queue)
+        if not item:
+            break
+        items.append(item)
+    return items
+
+
+def safe_get(queue):
+    '''
+    Get and return an item from the given queue.
+    If the queue is empty, reurn None (supressing the exception).
+    '''
+    try:
+        return (queue.get_nowait())
+    except Queue.Empty:
+        return
+    except Exception:
```

Comment:
This can be simplified to `except:`.

Reply:
https://github.com/AtomicConductor/conductor_client/pull/192#discussion_r181524105